examples/sparse/graphsage.py

```diff
@@ -0,0 +1,230 @@
+"""
+This script trains and tests a GraphSAGE model based on the information of 
```

Comment:
Please clarify what does this example demonstrate, emphasize the sparse sampling here.